src/Sulu/Component/Content/Tests/Unit/Extension/ExtensionManagerTest.php

```diff
@@ -125,10 +123,8 @@ public static function hasProvider()
         ];
     }
 
-    /**
-     * @dataProvider hasProvider
-     */
-    public function testHas($extensions, $type, $name, $expected): void
+    #[\PHPUnit\Framework\Attributes\DataProvider('hasProvider')]
+    public function testHas(array $extensions, string $type, string $name, bool $expected): void
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking change